bin/check_tron_jobs.py

```diff
@@ -153,24 +153,29 @@ def pretty_print_actions(action_run):
     return display.format_action_run_details(action_run)
 
 
-def get_relevant_run_and_state(job_runs):
-    if len(job_runs['runs']) == 0:
+def get_relevant_run_and_state(job_content):
+    job_runs = sorted(
+        job_content.get('runs', []),
+        key=lambda k: k['run_time'],
+        reverse=True,
+    )
+    if len(job_runs) == 0:
         return None, State.NO_RUN_YET
     run = is_job_scheduled(job_runs)
     if run is None:
-        return job_runs['runs'][0], State.NOT_SCHEDULED
-    run = is_job_stuck(job_runs)
+        return job_runs[0], State.NOT_SCHEDULED
+    run = is_job_stuck(job_runs, job_content.get('expected_runtime', None), job_content.get('actions_expected_runtime', {}))
```

Comment:
can you break this out and kwargs this please

Reply:
Sure, fixed.